src/main/scala/lms/transformation/distributed_tensor/distributedTensorSplit.scala

```diff
@@ -0,0 +1,109 @@
+package lms.transformation.tensor
+
+import scala.annotation.implicitNotFound
+import scala.collection._
+
+import lms.core._
+import lms.core.stub._
+import lms.collection.mutable._
+import lms.macros.SourceContext
+import lms.thirdparty.array_computation.{ArrayCPUOps, CUDATypeLess, CudaOps}
+
+import Backend._
+
+trait FixedSizeDistributedTensorSplitTypeLess extends FixedSizeDistributedTensorMutationTypeLess {
+  import BaseTypeLess._
+
+  class SPLIT_OP(override val x: Backend.Exp, override val useOldMetadata: Boolean = false) extends OPERATION(x, useOldMetadata) {
+    val axis: Int = gc.get(x.asInstanceOf[Backend.Sym]) match {
+      case Some(Node(_, "op_split", tts::anno::o::Backend.Const(axis:Int)::_, _)) => axis
+      case a => throw new Exception(s"Node $a is not a SplitOp")
+    }
+  }
+
+  def SplitOp(x: TENSOR, axis: Int, slices: List[Int], anno: Anno = NAnno)(implicit __pos: SourceContext): SPLIT_OP = {
```

Comment:
Nit: `SplitOp` -> `splitOp`

Reply:
I wanted to use `SplitOp` because I have `Add` `Sub` for other Tensors or Operations